ts/navigation/routes.ts

```diff
@@ -1,4 +1,8 @@
+import { Platform } from "react-native";
+
 const ROUTES = {
+  PREFIX: Platform.OS === "android" ? "italiaapp://italiaapp/" : "italiaapp://",
```

Comment:
add a comment explaining why this is different for Android and iOS

Reply:
Commented.